src/Model/CompanyResponse.php

```diff
@@ -49,6 +50,26 @@ final class CompanyResponse extends AbstractResponse
     #[SerializedName('previous_company_names')]
     public array $previousCompanyNames = [];
 
+    /**
+     * @var array<array-key, array{code: string, description: null|string}>
+     */
+    #[SerializedName('sic_codes')]
+    public array $sicCodes = [];
+
+    /**
+     * @param string[] $sicCodes
+     */
+    public function setSicCodes(array $sicCodes): void
+    {
+        $this->sicCodes = \array_map(
+            fn (string $code): array => [
+                'code'        => $code,
+                'description' => SicCodes::getDescriptionByCode($code),
+            ],
+            $sicCodes
+        );
+    }
```

Comment:
This would be perfect for a custom denormalizer ;)

Reply:
Yeah, I thought about it but thought this was easier :)